mautic/mautic
# Fix the issue with generating a report when a filter field value of a report is empty

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
When a field value of a filter of a report is empty, the error occurs. This PR fixes it.
![image](https://github.com/mautic/mautic/assets/96085911/06d4bb02-ab0e-4dd1-a71a-70732fefe012)
 


#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create a new report, e.g. contact report.
3. Add some any columns.
4. Add a filter and left the field empty.
5. Click the Save&Close button.
6. Works well. It was 500er before.
